src/color/shade.js

```diff
@@ -27,7 +27,7 @@ import mix from './mix'
 
 function shade(percentage: number | string, color: string): string {
   if (color === 'transparent') return color
-  return mix(parseFloat(percentage), 'rgb(0, 0, 0)', color)
+  return mix(parseFloat(percentage), '#000000', color)
```

Comment:
@thadeucity Any reason that we shouldn't send this as a shorthand '#000'?

Reply:
The the shorthand one will have two extra Regex validations and  two extra if validations (because the first check is for the normal 6 digit hex and the second one is for the 8 digit hex.